appcompat/appcompat/src/androidTest/java/androidx/appcompat/app/AppCompatDialogTest.kt

```diff
@@ -60,4 +60,32 @@ class AppCompatDialogTest {
                 .isNotSameInstanceAs(lifecycle)
         }
     }
+
+    @Test
+    fun testViewTreeLifecycleOwnerWhenAddContentView() {
+        withUse(ActivityScenario.launch(AppCompatActivity::class.java)) {
+            lateinit var view: View
+            val dialog = withActivity {
+                view = View(this)
+                AppCompatDialog(this)
+            }
+            dialog.addContentView(view)
```

Comment:
This doesn't compile since you're missing the `ViewGroup.LayoutParams` argument to `addContentView`.

Reply:
Sorry about that, my IDE can't compile the whole `androidx` project because of `internal Gradle distribution` issue.

```
The specified Gradle distribution 'file:/Users/chen.pan/Development/Workspace/AndroidStudioProjects/androidx/gradle/wrapper/../../../../tools/external/gradle/gradle-8.8-bin.zip' does not exist.
```